cmd/snap-remove.go

```diff
@@ -0,0 +1,142 @@
+/*
+ * MinIO Client (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/fatih/color"
+	"github.com/minio/cli"
+	json "github.com/minio/mc/pkg/colorjson"
+	"github.com/minio/mc/pkg/probe"
+	"github.com/minio/minio/pkg/console"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+var (
+	snapRemoveFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "force, f",
+			Usage: "Force removing the snapshot without a prompt",
```

Comment:
Do we really need to ask? It is 'only' a snapshot.

Just drop the confirmation IMO.

Reply:
> Do we really need to ask? It is 'only' a snapshot.
> 
> Just drop the confirmation IMO.

Let's wait until this is approved/disapproved by AB.

I always want confirmation for delete.

Reply:
`mc rm file` doesn't require confirmation. 

We are not deleting actual data, so I don't really see why it is so important.